subsys/emul/espi/emul_espi_host.c

```diff
@@ -147,10 +146,9 @@ static int emul_host_set_vw(struct espi_emul *emul, enum espi_vwire_signal vw,
 static int emul_host_get_vw(struct espi_emul *emul, enum espi_vwire_signal vw,
 			    uint8_t *level)
 {
-	struct espi_host_emul_data *data;
+	struct espi_host_emul_data *data = emul->parent->data;
```

Comment:
Why can't this accept the `struct emul *target`?

Reply:
Looks like I was lacking some changes in the headers as well as clients.

Done.